docroot/sites/now.uiowa.edu/modules/now_core/now_core.module

```diff
@@ -85,3 +138,65 @@ function now_core_node_presave($entity) {
     );
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function now_core_form_taxonomy_term_confirm_form_alter(&$form, FormStateInterface $form_state) {
+  // Check/prevent are auto-tags from being deleted on single delete.
```

Comment:
This sentence doesn't quite make sense to me.

Reply:
Doesn't make sense to me either. Updated to "  // Check/prevent tags from being deleted on single delete if they are in our list of tags used to associate article types."